### PR TITLE
Fix docstring for nn.Hardswish

### PR DESCRIPTION
**Pull request by lithuak**
Fixes nn.Hardswish's docstring problem reported at #70498.

### CODE REVIEW

File: torch/nn/modules/activation.py [hunk 1]
```diff
@@ -432,9 +432,10 @@ def extra_repr(self) -> str:
         return inplace_str
 
 class Hardswish(Module):
-    r"""Applies the hardswish function, element-wise, as described in the paper:
+    r"""Applies the Hardswish function, element-wise, as described in the paper:
+    `Searching for MobileNetV3 <https://arxiv.org/abs/1905.02244>`__.
```
Comment (jbschlosser) [inline torch/nn/modules/activation.py:436]: I think this should end in a single underscore, but I haven't seen the rendered docs yet

**Review by jbschlosser — COMMENTED** [review]
Thanks for the update! One possible typo but looks good otherwise. I do like being able to see paper links from the main activation function overview docs :)

File: torch/nn/modules/activation.py (hunk 1 above)
Comment (lithuak) [reply torch/nn/modules/activation.py]: you are right, I recompiled with "_" and still renders into a link (not surprisingly, as the docs say the same :)) )

**Comment by lithuak** [comment]
@jbschlosser yep, me to!

**Review by jbschlosser — APPROVED** [review]

**Pull request closed without merging**